sale_automatic_workflow_payment_mode/views/sale_workflow_process_view.xml

```diff
@@ -25,15 +25,15 @@ The licence is in the file __openerp__.py -->
                         name="payment_filter_domain"
```

Comment:
This should go also to the main module.

Reply:
Ok, some fields have been moved to the main addon `sale_automatic_workflow`.